pages/interface/components/Content/index.js

```diff
@@ -27,6 +27,16 @@ import {
 import { KebabHorizontalIcon, LinkIcon, PencilIcon, TrashIcon } from '@/TabNewsUI/icons';
 import { useUser } from 'pages/interface';
 
+const CONTENT_TITLE_PLACEHOLDER_EXAMPLES = [
+  'e.g. A nova era da computação quântica',
+  'e.g. Desafios ao empreender como desenvolvedor',
+  'e.g. Como funciona o conceito de ownership em Rust',
+  'e.g. Top 5 livros fundamentais para desenvolvedores',
```

Comment:
Passou pela minha cabeça agora, acho que essa frase fica melhor sem a palavra `Top` no início.